config/initializers/good_job.rb

```diff
@@ -0,0 +1,13 @@
+
+Rails.application.configure do
+  # Configure options individually...
+  config.good_job.preserve_job_records = true
+  # config.good_job.retry_on_unhandled_error = false
```

Comment:
you likely want to uncomment this and add
```
  # limit to 5 attempts
  retry_on StandardError, wait: :exponentially_longer, attempts: 5 do |_job, _exception|
    # Log error, do nothing, etc.
  end
```
to ApplicationJob to prevent infinite retry. See https://github.com/scientist-softserv/nnp/commit/ffc20e34aa1fd62de9cbef48e1d83084da5cae09

Reply:
That's, commit incoming...